### ISSUE TITLE
High DPI Version: Icons are very small

### ISSUE THREAD

**Issue by Hirogen** [zarunbal/LogExpert#246]
# Bug
High DPI Version: Icons are to small

Important questions
- LogExpert Version: 1.9.0 High DPI Version
- Windows Version: 11
- Lokal file or Network: NA
- Used Columnizer: NA
- Is this version specific or a new problem: High DPI Version problem

<img width="812" alt="image" src="https://user-images.githubusercontent.com/8225398/180168762-bc8222fc-fb9b-420e-9a5a-9ccbcbb4e425.png">

**Comment by Hirogen** [comment]
yeah true, currently looking in the open icon library but not all types of icons are included :/

**Comment by Hirogen** [comment]
I started to replace some of the icons, hopefully will be ready soon

**Comment by Hirogen** [comment]
Preview
<img width="332" alt="image" src="https://github.com/user-attachments/assets/9cda9bfb-18f4-4c85-9d59-01e198ea9a60" />

**Comment by TheNicker** [comment]
The icons Looks good.
Haven't tested it yet, but to my understanding, only ico format is full automatic DPI wise.
png might require minor manual tweaking.

I can test your changes if shared

**Comment by Hirogen** [comment]
yeah, but 48x48 pixel should be fine on most monitors, at least on my 4k monitor it looks fine :D

**Comment by TheNicker** [comment]
look good on 1440p

![Image](https://github.com/user-attachments/assets/3dce9a6d-a0f7-4eb2-ac2b-761715c34b69)

**Comment by Hirogen** [comment]
new version released: https://github.com/zarunbal/LogExpert/releases/tag/v.1.11.0
